### ISSUE TITLE
Feature Request: Run Program with a "Run As Administrator" option

### ISSUE THREAD

**Issue by brycebba** [rivafarabi/deckboard#301]
I run lots of programs as Admin that dont have start options to run as an admin. It would be great if there was an option in the Run Program task that was a Run As... or even a checkbox just to Run As Administrator.

**Comment by Yurlyn** [comment]
It'd have to mean Deckboard has to be run as administrator at least. But you could also go into whatever program's properties, then compatibility and then set the following (or similar option since I'm running Windows 7, lol):
![image](https://user-images.githubusercontent.com/42919979/68258038-b3523f80-0035-11ea-9e9a-5537c6bfd5c8.png)

**Comment by brycebba** [comment]
Right, I understand that would have to mean deckboard runs elevated which is fine with me for performing super user type actions. This would expand the usefulness of deckboard immensely.
